javalib/src/main/scala/java/net/InetAddress.scala

```diff
@@ -437,7 +438,7 @@ object InetAddress {
         val gaiMsg = SocketHelpers.getGaiErrorMessage(gaiStatus)
         val ex =
           if (gaiStatus == EAI_NONAME)
-            new UnknownHostException(gaiMsg)
+            new UnknownHostException(host + ": " + gaiMsg)
```

Comment:
To clarify: this is the only change I care about, since this was the regression.

I only made the other changes because they didn't make sense to me. Indeed the `unknownHostExceptionMsg()` method appeared to be effectively unused.

Reply:
I concur with the new line 441 and its inclusion of the host name.

The old line 440 looks like I intended to replace 440 with `unknownHostExceptionMst(host)` but missed it.
Arman, sorry for the mistake and it costing you time.

Reply:
I concur. Given my analysis below, adding `s"$host: $gaiMsg"` or using `host + ": " + gaiMsg`would match JVM as well.